components/telemetry-operator/go.mod

```diff
@@ -18,6 +18,7 @@ require (
 	k8s.io/api v0.25.3
 	k8s.io/apimachinery v0.25.3
 	k8s.io/client-go v0.25.3
+	k8s.io/klog/v2 v2.80.0
```

Comment:
Should be still an indirect dependency since it's only used by the logger package, right?

Reply:
Not entirely sure. I can move it to the indirect manually, it won't really change anything. However, when I do `go mod tidy`, it automatically moves it from indirect block back to the direct one